utils/generate_glossary.py

```diff
@@ -202,11 +202,6 @@ def generate_glossary(po_dir, output_path, input_glossary, output_glossary, only
             # We need shell=True for the wildcards.
             poterminology_result = subprocess.run(
                 ['poterminology ' + input_path + ' -o ' + pot_path], capture_output=True, shell=True, text=True, check=True)
-            if poterminology_result.returncode != 0:
```

Comment:
Where/How is this generate_glossary function used?

Reply:
This is unrelated. Somehow this slipped in the wrong commit, sorry.